webpack/scenes/Content/ContentConfig.js

```diff
@@ -0,0 +1,62 @@
+export function contentConfig() {
+  /*
+  [
+    {
+      key: 'python_package',
+      columnHeaders:
+      [
+        { title: __('Name'), getProperty: unit => unit?.name },
+        { title: __('Version'), getProperty: unit => unit?.version },
+      ],
+    },
+  ]
+  */
+  return [];
+}
+
+export function configuredTypes() {
+  const types = [];
+  contentConfig().forEach(type =>
+    types.push(type.names.title));
+  return types;
+}
+
+export function parseEnabledTypesResponse(enabledTypesResponse) {
+  const enabledTypes = {};
+  enabledTypesResponse.forEach((type) => {
+    const uiConfig = type.ui_config;
+    if (uiConfig) {
+      enabledTypes[uiConfig.names.title] = {
+        labels: [uiConfig.names.singular, uiConfig.names.plural],
+        uiConfig,
+      };
+    }
+  });
+  return enabledTypes;
+}
+
+export function getConfiguredTypes(enabledTypes) {
+  const types = [];
+  Object.entries(enabledTypes).forEach(([key, value]) => {
+    if (configuredTypes().includes(value[0])) {
+      types.push(key);
+    }
+  });
+  return types;
```

Comment:
What is this doing exactly?

Reply:
It's checking the enabled content types against what has a configuration defined in `ContentConfig()`. It returns a list of keys for content types that are enabled _and_ have a configuration specified. If a content type is in this list, it won't use `DefaultConfig()`.

It might benefit from a different name.

Reply:
I wasn't able to get it working with an override config locally..Can you verify that it works for you and let me know the config you used?